content/2. Ondersteunende-informatie/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Integer/1. Integer.md

```diff
@@ -0,0 +1,61 @@
+---
+title: 1. Integer
+taxonomie:
+  - ib-19.2.Datatype-Primitief-Integer.OI
+  - ib-19.3.Datatype-Primitief-Integer.OI
+---
+
+## Wat is een Integer?
+Een Integer is een datatype waar een positief of negatief getal in past. Een programmeur kan dit datatype aan een variabel geven en hiermee getallen bewaren, bewerken en uitlezen.
```

Comment:
Opmerking: dit is afhankelijk van of het een signed of unsigned integer is (als in, het ene laat alleen positief toe, de ander beiden). Wellicht handig om kort beiden te benoemen, en tussen haakjes achter positief/negatief te zetten om welke het gaat?



Edit: ah, ik zie dat uint beneden wordt behandeld. In dat geval, misschien een verwijzing hiernaar? (iets als zie uint).

Reply:
Ik heb hier "Een Signed Integer, vaak benoemd als Integer, is een datatype..." van gemaakt met een verwijzing naar Uint aan het einde van de alinea. Is dit een beetje wat je in gedachten had?